Cargo.toml

```diff
@@ -24,4 +24,6 @@ polling = "2.1"                                        # select/poll sockets
 socket2 = { version = "0.5.5", features = ["all"] }      # socket APIs
 
 [dev-dependencies]
+env_logger = "0.11.3"
 fastrand = "1.8"
+test-log = "0.2.15"
```

Comment:
First time seeing this crate. Seems neat based on its doc. Is it working well for you?

Reply:
This one was tricky to downgrade for some reason. Cargo kept insisting that we use version 0.2.15 of transitive dependency `test-log-macros`, despite `test-log 0.2.14` depending on version `test-log-macros 0.2.14` itself. I've included `test-log-macros` as a dev dependency just to trick cargo to use the right one, but we never use that dependency directly.

I have not used this crate before myself, but I've found myself adding `env_logger::init()` to the start of every single test I write. I'll probably start using it from now on.